autocommit.js

```diff
@@ -63,7 +63,8 @@ openai
     const commitMessage = data.data.choices[0].text;
     const command = `git add --all && git commit -m "${commitMessage.replace(/"/g, '\\"')}"`;
 
-    if (!process.env.GIT_AI_AUTOCOMMIT) {
+    // Check if process.env.GIT_AI_AUTOCOMMIT exists and is set to 1 or true, otherwise just warn.
+    if (!(process.env.GIT_AI_AUTOCOMMIT && process.env.GIT_AI_AUTOCOMMIT.match(/^(1|true)$/))) {
```

Comment:
swap if/else statements to remove `!()` part
and make it case insensitive: `/^(1|true)$/i)`